packages/module/src/ConsoleShared/src/components/layout/PageLayout.scss

```diff
@@ -1,4 +1,4 @@
-.ocs-page-layout {
```

Comment:
Anything with the `ocs-` prefix is specific to OpenShift Container Storage and shouldn't be related to quick starts. Are we sure these should have been copied over to this repo? Can we remove any of these classes rather than just renaming?

Reply:
The ocs styles had been in use for some of the catalog components, for example https://github.com/patternfly/patternfly-quickstarts/blob/0b6068f9a1654698175ec5de70becded75dfde11/packages/module/src/QuickStartCatalogPage.tsx#L143